src/App.js

```diff
@@ -24,50 +24,42 @@ import CheckoutForm from "./components/checkout/CheckoutForm";
 import Return from "./components/checkout/CheckoutReturn";
 import PricingPage from "./components/checkout/PricingPage";
 import { IntlProvider } from "react-intl";
+import Navbar2 from "./components/newUIComponents/nav/Navbar2";
+import { newTheme, oldTheme } from "./themes";
+import Home2 from "./components/newUIComponents/Home2";
+import Footer2 from "./components/newUIComponents/Footer2";
 import Mapping from "./components/mapping/Mapping";
 
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  TimeScale,
+} from "chart.js";
+
+import "chartjs-adapter-moment";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  TimeScale,
+);
+
 Amplify.configure(awsExports);
 function App() {
-  const theme = {
```

Comment:
This theme is used for the amplify login presented.  It's still necessary in some form to make sure that renders like the rest of the app (even if updated to account for different font, etc).
![Screenshot 2023-12-09 at 10 56 18 AM](https://github.com/bigboxer23/solar-moon-frontend/assets/716472/4ae9b3af-4739-4cbf-9be6-77c1c7aa4bb1)

Reply:
nm, i see its just moved to external file, though the styling on new theme is still off.